doc/src/getting-started-guide.md

```diff
@@ -209,11 +209,154 @@ Once the application has been flashed, you may connect to the ESP32 over the ser
 
 ## Getting Started on the STM32 platform
 
-> TODO will se distribute a binary image for STM32?
+AtomVM can run on a wide variety of STM32 chipsets available from [STMicroelectronics](https://www.st.com). The support is not nearly as mature as for the ESP32 platform, but work is ongoing, and pull requests are always welcome. At this time AtomVM will work on any board with a minimum of around 128k ram and 512k (1M recommended) flash. Simple applications and tests have been successfully run on a stm32f411ceu6 (A.K.A. Black Pill V2). These minimum requirements may need to be raised as platform support matures.
+
+### Prerequisites
+
+* [st-flash](https://github.com/texane/stlink), to flash both AtomVM and your packed AVM applications. Make sure to follow its [installation procedure](https://github.com/texane/stlink#installation) before proceeding further.
+* arm-none-eabi/arm-elf toolchain compatible with your system
+* `cmake`
+* `make`
+* A serial console program, such as `minicom` or `screen`, so that you can view console output from your AtomVM application.
+
+### Building
+Before building for the first time you need to have a compiled clone of the libopencm3 libraries, from inside the AtomVM/src/platforms/stm32 directory:
+
+    $ git clone https://github.com/libopencm3/libopencm3.git
+    $ cd libopencm3 && make -j4 && cd ..
+    $ mkdir build
+    $ cd build
+    $ cmake -DCMAKE_TOOLCHAIN_FILE=../cmake/arm-toolchain.cmake
+    $ make
+
+You can put libopencm3 wherever you want on your PC as long as you update LIBOPENCM3_DIR to point to it. This example assumes it has been cloned into /opt/libopencm3 and built. From inside the AtomVM/src/platforms/stm32 directory:
+
+	$ mkdir build
+	$ cd build
+    $ cmake -DCMAKE_TOOLCHAIN_FILE=../cmake/arm-toolchain.cmake -DLIBOPENCM3_DIR=/opt/libopencm3 ..
+    $ make
+
+### Flashing
+To flash AtomVM, use
+
+    $ st-flash --reset write atom-vm.bin 0x8000000
+
+To flash your packed AVM, use
+
+    $ st-flash --reset write /path/to/your/packed.avm 0x8080000
+
+You must include the atomvmlib.avm with your application when using packbeam, and it should be pruned:
+
+	$ packbeam create -p -i application.avm application.beam /path/to/AtomVM/build/libs/atomvmlib.avm
```

Comment:
We would need to reference the atomvm_packbeam tool here.

Reply:
Yes! I had planned on that, but forgot to go back and add a link.